buildpacks/jvm/src/bin/heroku_rewrite_env_vars.rs

```diff
@@ -624,4 +642,90 @@ mod tests {
 
         assert_eq!(result.get("SPRING_REDIS_URL"), None);
     }
+
+    #[test]
```

Comment:
@Malax Is there an integration test for any of the env var mappings?

While the unit test does 95% of the job, a single integration test (perhaps as part of another test, to reduce end to end time) is needed to check that the binary is actually wired up too.